webapp/channels/src/components/suggestion/switch_channel_provider.tsx

```diff
@@ -838,12 +838,12 @@ export default class SwitchChannelProvider extends Provider {
         if (!teamId) {
             return;
         }
-        const channelsAsync = this.store.dispatch(fetchAllMyTeamsChannelsAndChannelMembersREST());
+        const channelsAsync = this.store.dispatch(fetchAllMyTeamsChannels());
```

Comment:
Just to note that this is a functional change compared to the previous version of the code. I don't think it's likely to matter since it looks like this code only triggers if this modal somehow appears if there are no recent channels loaded which seems impossible

Reply:
Yep, I'm pretty sure it only needs to care about channels, but even if not if this loading is done passively and is required by this component (or some other for some reason), it will eventually get loaded by `team_controller`. If we find this reveals a bug, it was probably always there anyways and was just masked by the fact that we aggressively loaded the channel members, apparently multiple times.